koiki/delivery.py

```diff
@@ -9,3 +14,12 @@ def to_dict(self):
             'barcode': self.data['codBarras'],
             'label': self.data['etiqueta']
         }
+
+    def print_pdf(self):
+        content = base64.b64decode(self.data['etiqueta'])
+
+        pdf = open(self.filename, 'wb')
```

Comment:
Could this also be mocked somehow?, because during tests, it generates me a label.pdf every time.

Reply:
I thought it wasn't a big deal for now but yes, you can mock `print_pdf`.

Reply:
Is still not big deal xD